Makefile

```diff
@@ -82,7 +82,10 @@ cabal.project.local:
 
 # Usage: make c package=brig test=1
 .PHONY: c
-c: treefmt
+c: treefmt c-fast
```

Comment:
Now that I see this, I'm wondering if our approach to squeeze everything into one makefile target fits to the idea of `make`. I.e. if someone wants to compile and format in one go, why don't they call e.g. `make c format`? :thinking: 

(I'm fine with your change though, because your intention probably was to keep the dev's interface stable.)